docs/journal/01-cold-storage.md

```diff
@@ -76,14 +92,13 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 - **Throughput.** A worker role has less overhead than a web role (*NEED SOMEONE TO VERIFY THIS, AND MORE DETAIL ON WHY*)
 
-- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing the data for an event this data is not lost but is instead returned to the Service Bus topic from whence it can be retrieved again.
+- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing an event, the event is returned to the Service Bus topic and can be retrieved again.
```

Comment:
That's my concern. I know that AMQP has many features that are not relevant to something like Event Hubs (such as routing). My understanding is that the client libraries implement a subset of the features. I don't know where reliability fits in there.
In the .NET library for Event Hub, I believe that retries are handled higher in the stack (so that HTTP is also supported).